docs/en/compute/Presto.md

```diff
@@ -212,12 +212,6 @@ For Alluxio 1.6 or earlier,
 it is recommended to set this size no less than Alluxio's block size to avoid the read contention within the same block. For later Alluxio versions, this is no more an issue due to
 async cache on Alluxio workers.
 
-### Avoid Presto timeout reading large files
-
-It is recommended to increase `alluxio.user.network.netty.timeout` to a bigger value (e.g.
```

Comment:
@bf8086  do you have any recommendation here after  `alluxio.user.network.netty.timeout` is removed in 2.0?

Reply:
The equivalent property in Alluxio 2.0 should be `alluxio.user.network.data.timeout`.